doc/man7/OSSL_PROVIDER-default.pod

```diff
@@ -265,6 +267,8 @@ The OpenSSL default provider supports these operations and algorithms:
 
 =item SM2, see L<EVP_KEYMGMT-SM2(7)>
 
+=item ML-KEM-768, see L<EVP_KEYMGMT-ML-KEM(7)>
```

Comment:
```suggestion
=item ML-KEM-768, see L<EVP_KEYMGMT-ML-KEM-768(7)>
```

Reply:
Changed to all specific key types in https://github.com/openssl/openssl/pull/26037/commits/5b7aad109796393b11b35d43474b444283ccb698